### PR TITLE
#19 there should be a document describing justaskdavidbs coaching program

### PR DESCRIPTION
**Pull request by justaskdavidb**

### CODE REVIEW

**Review by langrh — COMMENTED** [review]
I like this, I feel like I got a great understanding of what the JustAskDavidB coaching program is all about, what is offered, and who the program caters to. This is great.

**Comment by justaskdavidb** [comment]
@KellynBE , do you mind reviewing and approving this?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications
```
Comment (KellynBE) [inline README.md:1]: Is this the actual title? If so, doesn't need to be hyphenated and should be capitalized

File: README.md [hunk 2]
```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme.
```
Comment (KellynBE) [inline README.md:3]: - The / should also be a comma
- change "opportunity" to "opportunities"
- comma before "which"
- comma after "etc."

File: README.md [hunk 3]
```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
+
+## Purpose  
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
```
Comment (KellynBE) [inline README.md:7]: - delete "s" in "fives"

File: README.md [hunk 4]
```diff
@@ -1,2 +1,58 @@
...
+- individuals zero prior IT experience
```
Comment (KellynBE) [inline README.md:8]: - capitalize first word of all bulleted items
- insert "with" after "individuals"

File: README.md [hunk 5]
```diff
@@ -1,2 +1,58 @@
...
+# indevelopme-coaching-publications  
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
+
+## Purpose  
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
+- individuals zero prior IT experience  
+- individuals pursuing an IT degree  
+- individuals seek a career shift into IT
```
Comment (KellynBE) [inline README.md:10]: "seek" should be "seeking"

File: README.md [hunk 6]
```diff
@@ -1,2 +1,58 @@
...
+
+This repository holds a free collection of IT knowledge that has been created and reviewed for publication into Medium / Linkedin, and other platforms. Participants in the JustAskDavidB Coaching Program (aka GSD-IT) maintain this repository to showcase their IT compentency with hopes of improving IT literacy in non-IT communities. Highly consistent contributors have a chance to be offered paid opportunity which includes free self-training products such as Linkedin Learning, Coursera, etc. and opportunities to work on paid projects for inDevelopme. 
+
+## Purpose  
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
+- individuals zero prior IT experience  
+- individuals pursuing an IT degree  
+- individuals seek a career shift into IT  
+- mid and senior level IT professions trying to enter management
```
Comment (KellynBE) [inline README.md:11]: - hyphen after "mid"
- hyphenate "senior-level"
- change "professions" to "professionals"

File: README.md [hunk 7]
```diff
@@ -1,2 +1,58 @@
...
+
+The JustAskDavidB Coaching Program intends to help fives groups of people develop understanding of IT concepts at an accelerated rate:
+- individuals zero prior IT experience  
+- individuals pursuing an IT degree  
+- individuals seek a career shift into IT  
+- mid and senior level IT professions trying to enter management  
+
+## Description  
+
+Participants will work under the guidance of a coach to complete open-source initiatives for inDevelopme's, with tasks being specific to the individual's selected coaching track (e.g. Programming, DevOps, Content Strategy). When necessary, participants shall be granted access to software licenses, credentails, and information to complete tasks.
```
Comment (KellynBE) [inline README.md:15]: - "inDevelopme" not "inDevelopme's"
- comma after e.g.
- lowercase the tracks

File: README.md [hunk 8]
```diff
@@ -1,2 +1,58 @@
...
+
+Participants will work through a series of tasks meticulously designed to foster growth in programming languages (e.g. Python, Java, C++, etc). They will tackle debugging scenarios, conditional statements, iterative programming,practice user input and output handling, and build basic user interfaces. Other tasks will include self-exploration in data structures, algorithms, version control, etc to participant on small projects designed to allow application of newly acquired knowledge and skills.  
+
+***Programming: Benefits to Participant***  
+
+Consistent engagement in programming will help partcipants build a solid foundation in programming, enabling them to take on more complex challenges. Participants will also develop a portfolio of skills and work experience for the world to see!
+
+### DevOps  
+
+This GSD-IT track focuses heavily on devops concepts and best practices using AWS. Docker, Jenkins, Terraform, and Amazon Web Service (AWS) are just some of the products and technologies used within this track.
```
Comment (KellynBE) [inline README.md:38]: - should be "DevOps"
- spell out AWS on first usage with AWS in parentheses, and then use AWS on second reference

File: README.md [hunk 9]
```diff
@@ -1,2 +1,58 @@
...
+
+***DevOps: General Tasks***  
+
+Participants will apply fundamental concepts and utilize essentials tools such as Jenkins, Docker, and AWS. The will learn to use and create IaaS, PaaS, and SaaS products and services. Participants will explore concepts such as Infrastructure as Code (IaC) & Continous Integration / Continuous Deployment (CI/CD) to complete tasks for inDevelopme's open-source libraries. A participant may be asked to learn about containerizaiton or container orchestration tools -- and then be asked to demonstration competency through an implementation. This has a heavy focus in sucrity best practices, collaboration, application architecture, and documentation.
```
Comment (KellynBE) [inline README.md:42]: - second sentence, "The" should be "They"
- What's laaS, PaaS, and SaaS? Should it be LaaS?
- Change the & for "and"
- no spaces around /
- correct spelling of containerization
- reword - "orchestration tools, and then to demonstrate"
- change "demonstration" to "demonstrate"
- is "sucrity" supposed to be "security"?

File: README.md [hunk 10]
```diff
@@ -1,2 +1,58 @@
...
+
+This GSD-IT track focuses heavily on devops concepts and best practices using AWS. Docker, Jenkins, Terraform, and Amazon Web Service (AWS) are just some of the products and technologies used within this track.
+
+***DevOps: General Tasks***  
+
+Participants will apply fundamental concepts and utilize essentials tools such as Jenkins, Docker, and AWS. The will learn to use and create IaaS, PaaS, and SaaS products and services. Participants will explore concepts such as Infrastructure as Code (IaC) & Continous Integration / Continuous Deployment (CI/CD) to complete tasks for inDevelopme's open-source libraries. A participant may be asked to learn about containerizaiton or container orchestration tools -- and then be asked to demonstration competency through an implementation. This has a heavy focus in sucrity best practices, collaboration, application architecture, and documentation. 
+  
+***DevOps: Benefits to Participant***  
+
+Consistent engagement in the devops track will help partcipants build a solid foundation in cloud architecture. Participants will develop skill that help solve high availability, reliability, and scalability issues within an cloud application.
```
Comment (KellynBE) [inline README.md:46]: - should be DevOps
- add an s to "skill"
- change "an" to "a" (a cloud application"

File: README.md [hunk 11]
```diff
@@ -1,2 +1,58 @@
...
+  
+***DevOps: Benefits to Participant***  
+
+Consistent engagement in the devops track will help partcipants build a solid foundation in cloud architecture. Participants will develop skill that help solve high availability, reliability, and scalability issues within an cloud application. 
+
+### Content Strategy
+
+This GSD-IT track focuses heavily content development best practices in the digital space. This track uses technologies and concepts such as Squarespace, Stripe, Medium, Google Analytics, etc to teach participants IT concepts around website development. This is done to help participants in the Business Administration and Fine Arts field better understand the role that IT plays in the content development life cycle.
```
Comment (KellynBE) [inline README.md:50]: - insert "on" after "focuses"
- add period and comma after "etc"
- lowercase Business Administration and Fine Arts

File: README.md [hunk 12]
```diff
@@ -1,2 +1,58 @@
...
+  
+***Content Strategy: General Tasks***  
+
+Participants on the Content Strategy track will complete tasks related to the content development for inDevelopme's non-profit initiatives. This will include the development of style guides, voice and tone, user personas, etc for inDevelopme as outlined by inDevelopme's Content Playbook. Participants will also utilize Search Engine Optimization (SEO) and Marketing (SEM) best practices to raise content engagement for JustAskDavidB and other inDevelopme non-profit initiatives.
```
Comment (KellynBE) [inline README.md:54]: - no hyphen in nonprofit (2 instances)
- add period and comma after "etc"

File: README.md [hunk 13]
```diff
@@ -1,2 +1,58 @@
...
+
+***Content Strategy: Benefit to Participant***  
+
+Consistent engagement in this track will establish solid foundation in the content developemnt lifecycle and general IT concepts such "development environments" and User Acceptance Testing (UAT). Participants will occassional have working sessions with content strategist that have 15+ years of experience in the field. These opportunities will help participants become proficient at making strategic decisions backed by research and data for the purpose of increasing content engagement.
```
Comment (KellynBE) [inline README.md:58]: - insert "a" before "solid foundation"
- "life cycle" is two words
- why is development environments in quotations but not UAT?
- change "occasional" to "occasionally" 
- make "strategist" plural

**Review by KellynBE — CHANGES_REQUESTED** [review]

File: README.md [hunk 14]
```diff
@@ -1,2 +1,58 @@
...
+## Tracks 
+
+Participants can choose to receive guidance from 1 of 3 GSD-IT categories:
+- Programming
+- DevOps
+- Content Strategy
+
+### Programming
+
+This GSD-IT track focuses heavily on programming concepts and best practicing. GIT, Composer, Jetbrain, Laravel (PHP), and Flask (Python) represent some of the products and technologies used within this track.
```
[the remark this answers is not in the document]
Comment (justaskdavidb) [reply README.md:26]: Yes, Git is the technology used to support GitHub.  [ Git](https://git-scm.com/book/en/v2/Getting-Started-A-Short-History-of-Git) was created 2005 and [GitHub](https://en.wikipedia.org/wiki/GitHub) started in 2007.

File: README.md (hunk 9 above)
Comment (justaskdavidb) [reply README.md:42]: [IaaS](https://www.techtarget.com/searchcloudcomputing/definition/Infrastructure-as-a-Service-IaaS)
[PaaS](https://cloud.google.com/learn/what-is-paas)
[SaaS](https://www.salesforce.com/in/saas)

Comment (justaskdavidb) [reply README.md:42]: yep meant "security"

File: README.md [hunk 15]
```diff
@@ -1,2 +1,58 @@
...
+Participants can choose to receive guidance from 1 of 3 GSD-IT categories:
+- Programming
+- DevOps
+- Content Strategy
+
+### Programming
+
+This GSD-IT track focuses heavily on programming concepts and best practicing. GIT, Composer, Jetbrain, Laravel (PHP), and Flask (Python) represent some of the products and technologies used within this track. 
+
+***Programming: General Tasks***
```
[the remark this answers is not in the document]
Comment (justaskdavidb) [reply README.md:28]: Yea I know, but I wanted to make it easy for someone to jump specifically the parts they care about.

**Review by justaskdavidb — COMMENTED** [review]
@KellynBE - this is ready for medium now.

**Review by KellynBE — APPROVED** [review]

**Pull request merged**